# Descriptive Statistics for Total Offense Points for each team

#### Import packages

In [1]:
import pandas as pd
import numpy as np
from scipy.stats import norm
from scipy import stats

#### Import NFL Dataset 2012-2022

In [3]:
nflstats = pd.read_csv('/Users/admin/Desktop/Final_Project/NFL_merged/nfl2012_2022merge.csv')

In [4]:
nflstats.head()

Team  Off Tot/Pts  Off Pts/G  Off RushYds  Off RYds/G  \
0  Arizona Cardinals          250       15.6         1204        75.2   
1    Atlanta Falcons          419       26.2         1397        87.3   
2   Baltimore Ravens          398       24.9         1901       118.8   
3      Buffalo Bills          344       21.5         2217       138.6   
4  Carolina Panthers          357       22.3         2088       130.5   

   Off PassYds  Off PYds/G  Off TotYds  Off Yds/G  Def Tot/Pts  ...  \
0         3005       187.8        4209      263.1          357  ...   
1         4509       281.8        5906      369.1          299  ...   
2         3739       233.7        5640      352.5          344  ...   
3         3269       204.3        5486      342.9          435  ...   
4         3683       230.2        5771      360.7          363  ...   

   Def PassYds  Def PYds/G  Def TotYds  Def Yds/G  Year  Wins  Losses  Tie  \
0         3213       200.8        5405      337.8  2012     5      11  0.0   
1         3878       242.4        5849      365.6  2012    13       3  0.0   
2         3650       228.1        5615      350.9  2012    10       6  0.0   
3         3473       217.1        5806      362.9  2012     6      10  0.0   
4         3568       223.0        5329      333.1  2012     7       9  0.0   

   Superbowl  SB Win  
0         No      No  
1         No      No  
2        Yes     Yes  
3         No      No  
4         No      No  

[5 rows x 23 columns]

## Calculating Descriptive Statistics by Team
#### First filter the data by a team

In [5]:
team_name = 'Seattle Seahawks'
seahawks_data = nflstats[nflstats['Team'] == team_name]

#### Display each total offense points by year for the Seahawks

In [6]:
for year in seahawks_data['Year'].unique():
    offense_points = seahawks_data.loc[seahawks_data['Year'] == year, 'Off Tot/Pts'].values[0]
    print("Year:", year, "\tOffense Points:", offense_points)

Year: 2012 	Offense Points: 412
Year: 2013 	Offense Points: 417
Year: 2014 	Offense Points: 394
Year: 2015 	Offense Points: 423
Year: 2016 	Offense Points: 354
Year: 2017 	Offense Points: 366
Year: 2018 	Offense Points: 366
Year: 2019 	Offense Points: 405
Year: 2020 	Offense Points: 459
Year: 2021 	Offense Points: 395
Year: 2022 	Offense Points: 407


#### What is the mean total offense pts for all years for the Seahawks?

In [8]:
total_offense_points = seahawks_data['Off Tot/Pts'].mean()

In [9]:
print(total_offense_points)

399.8181818181818


#### Let's calculate several descriptive statistics on Offense Total Pts for the Seahawks

In [102]:
mean_value = seahawks_data['Off Tot/Pts'].mean()
median_value = seahawks_data['Off Tot/Pts'].median()
std_value = seahawks_data['Off Tot/Pts'].std()
var_value = seahawks_data['Off Tot/Pts'].var()
min_value = seahawks_data['Off Tot/Pts'].min()
max_value = seahawks_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = seahawks_data['Off Tot/Pts'].sum()
print("Seattle Seahawks Offense Points Mean:", mean_value)
print("Seattle Seahawks Offense Points Median:", median_value)
print("Seattle Seahawks Offense Points Standard Deviation:", std_value)
print("Seattle Seahawks Offense Points Variance:", var_value)
print("Seattle Seahawks Offense Points Minimum:", min_value)
print("Seattle Seahawks Offense Points Maximum:", max_value)
print("Seattle Seahawks Offense Points Range:", data_range)
print("Seattle Seahawks Total Points:", total_points)

Seattle Seahawks Offense Points Mean: 399.8181818181818
Seattle Seahawks Offense Points Median: 405.0
Seattle Seahawks Offense Points Standard Deviation: 30.009392469086016
Seattle Seahawks Offense Points Variance: 900.5636363636364
Seattle Seahawks Offense Points Minimum: 354
Seattle Seahawks Offense Points Maximum: 459
Seattle Seahawks Offense Points Range: 105
Seattle Seahawks Total Points: 4398


#### Calculating all Teams at total offense points per year

In [12]:
team_names = nflstats['Team'].unique()

# Iterate over each team
for team_name in team_names:
    team_data = nflstats[nflstats['Team'] == team_name]
    
    # Display the total offense points for each year
    for year in team_data['Year'].unique():
        offense_points = team_data.loc[team_data['Year'] == year, 'Off Tot/Pts'].values[0]
        print("Team:", team_name, "\tYear:", year, "\tTotal Offense Points:", offense_points)

Team: Arizona Cardinals 	Year: 2012 	Total Offense Points: 250
Team: Arizona Cardinals 	Year: 2013 	Total Offense Points: 379
Team: Arizona Cardinals 	Year: 2014 	Total Offense Points: 310
Team: Arizona Cardinals 	Year: 2015 	Total Offense Points: 489
Team: Arizona Cardinals 	Year: 2016 	Total Offense Points: 418
Team: Arizona Cardinals 	Year: 2017 	Total Offense Points: 295
Team: Arizona Cardinals 	Year: 2018 	Total Offense Points: 295
Team: Arizona Cardinals 	Year: 2019 	Total Offense Points: 361
Team: Arizona Cardinals 	Year: 2020 	Total Offense Points: 410
Team: Arizona Cardinals 	Year: 2021 	Total Offense Points: 449
Team: Arizona Cardinals 	Year: 2022 	Total Offense Points: 340
Team: Atlanta Falcons 	Year: 2012 	Total Offense Points: 419
Team: Atlanta Falcons 	Year: 2013 	Total Offense Points: 353
Team: Atlanta Falcons 	Year: 2014 	Total Offense Points: 381
Team: Atlanta Falcons 	Year: 2015 	Total Offense Points: 339
Team: Atlanta Falcons 	Year: 2016 	Total Offense Points: 540
Te

#### Calculate descriptive statistics on Offense Total Points for all teams. These statistics are the Total Offense Points and their mean, median, standard deviation, variance, min, max, range, and sum.
### Arizona Cardinals

In [101]:
team_name1 = 'Arizona Cardinals'
cardinals_data = nflstats[nflstats['Team'] == team_name1]
mean_value = cardinals_data['Off Tot/Pts'].mean()
median_value = cardinals_data['Off Tot/Pts'].median()
std_value = cardinals_data['Off Tot/Pts'].std()
var_value = cardinals_data['Off Tot/Pts'].var()
min_value = cardinals_data['Off Tot/Pts'].min()
max_value = cardinals_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = cardinals_data['Off Tot/Pts'].sum()
print("Arizona Cardinals Offense Points Mean:", mean_value)
print("Arizona Cardinals Offense Points Median:", median_value)
print("Arizona Cardinals Offense Points Standard Deviation:", std_value)
print("Arizona Cardinals Offense Points Variance:", var_value)
print("Arizona Cardinals Offense Points Minimum:", min_value)
print("Arizona Cardinals Offense Points Maximum:", max_value)
print("Arizona Cardinals Offense Points Range:", data_range)
print("Arizona Cardinals Total Points:", total_points)

Arizona Cardinals Offense Points Mean: 363.27272727272725
Arizona Cardinals Offense Points Median: 361.0
Arizona Cardinals Offense Points Standard Deviation: 73.56642020526881
Arizona Cardinals Offense Points Variance: 5412.018181818183
Arizona Cardinals Offense Points Minimum: 250
Arizona Cardinals Offense Points Maximum: 489
Arizona Cardinals Offense Points Range: 239
Arizona Cardinals Total Points: 3996


### Atlanta Falcons

In [100]:
team_name2 = 'Atlanta Falcons'
falcons_data = nflstats[nflstats['Team'] == team_name2]
mean_value = falcons_data['Off Tot/Pts'].mean()
median_value = falcons_data['Off Tot/Pts'].median()
std_value = falcons_data['Off Tot/Pts'].std()
var_value = falcons_data['Off Tot/Pts'].var()
min_value = falcons_data['Off Tot/Pts'].min()
max_value = falcons_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = falcons_data['Off Tot/Pts'].sum()
print("Atlanta Falcons Offense Points Mean:", mean_value)
print("Atlanta Falcons Offense Points Median:", median_value)
print("Atlanta Falcons Offense Points Standard Deviation:", std_value)
print("Atlanta Falcons Offense Points Variance:", var_value)
print("Atlanta Falcons Offense Points Minimum:", min_value)
print("Atlanta Falcons Offense Points Maximum:", max_value)
print("Atlanta Falcons Offense Points Range:", data_range)
print("Atlanta Falcons Total Points:", total_points)

Atlanta Falcons Offense Points Mean: 381.1818181818182
Atlanta Falcons Offense Points Median: 365.0
Atlanta Falcons Offense Points Standard Deviation: 59.954679853733154
Atlanta Falcons Offense Points Variance: 3594.563636363636
Atlanta Falcons Offense Points Minimum: 313
Atlanta Falcons Offense Points Maximum: 540
Atlanta Falcons Offense Points Range: 227
Atlanta Falcons Total Points: 4193


### Baltimore Ravens

In [99]:
team_name3 = 'Baltimore Ravens'
ravens_data = nflstats[nflstats['Team'] == team_name3]
mean_value = ravens_data['Off Tot/Pts'].mean()
median_value = ravens_data['Off Tot/Pts'].median()
std_value = ravens_data['Off Tot/Pts'].std()
var_value = ravens_data['Off Tot/Pts'].var()
min_value = ravens_data['Off Tot/Pts'].min()
max_value = ravens_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = ravens_data['Off Tot/Pts'].sum()
print("Baltimore Ravens Offense Points Mean:", mean_value)
print("Baltimore Ravens Offense Points Median:", median_value)
print("Baltimore Ravens Offense Points Standard Deviation:", std_value)
print("Baltimore Ravens Offense Points Variance:", var_value)
print("Baltimore Ravens Offense Points Minimum:", min_value)
print("Baltimore Ravens Offense Points Maximum:", max_value)
print("Baltimore Ravens Offense Points Range:", data_range)
print("Baltimore Ravens Total Points:", total_points)

Baltimore Ravens Offense Points Mean: 393.09090909090907
Baltimore Ravens Offense Points Median: 395.0
Baltimore Ravens Offense Points Standard Deviation: 62.36738658217858
Baltimore Ravens Offense Points Variance: 3889.690909090909
Baltimore Ravens Offense Points Minimum: 320
Baltimore Ravens Offense Points Maximum: 531
Baltimore Ravens Offense Points Range: 211
Baltimore Ravens Total Points: 4324


### Buffalo Bills

In [98]:
team_name4 = 'Buffalo Bills'
bills_data = nflstats[nflstats['Team'] == team_name4]
mean_value = bills_data['Off Tot/Pts'].mean()
median_value = bills_data['Off Tot/Pts'].median()
std_value = bills_data['Off Tot/Pts'].std()
var_value = bills_data['Off Tot/Pts'].var()
min_value = bills_data['Off Tot/Pts'].min()
max_value = bills_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = bills_data['Off Tot/Pts'].sum()
print("Buffalo Bills Offense Points Mean:", mean_value)
print("Buffalo Bills Offense Points Median:", median_value)
print("Buffalo Bills Offense Points Standard Deviation:", std_value)
print("Buffalo Bills Offense Points Variance:", var_value)
print("Buffalo Bills Offense Points Minimum:", min_value)
print("Buffalo Bills Offense Points Maximum:", max_value)
print("Buffalo Bills Offense Points Range:", data_range)
print("Buffalo Bills Total Points:", total_points)

Buffalo Bills Offense Points Mean: 378.27272727272725
Buffalo Bills Offense Points Median: 344.0
Buffalo Bills Offense Points Standard Deviation: 72.16244301448074
Buffalo Bills Offense Points Variance: 5207.418181818181
Buffalo Bills Offense Points Minimum: 302
Buffalo Bills Offense Points Maximum: 501
Buffalo Bills Offense Points Range: 199
Buffalo Bills Total Points: 4161


### Carolina Panthers

In [97]:
team_name5 = 'Carolina Panthers'
panthers_data = nflstats[nflstats['Team'] == team_name5]
mean_value = panthers_data['Off Tot/Pts'].mean()
median_value = panthers_data['Off Tot/Pts'].median()
std_value = panthers_data['Off Tot/Pts'].std()
var_value = panthers_data['Off Tot/Pts'].var()
min_value = panthers_data['Off Tot/Pts'].min()
max_value = panthers_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = panthers_data['Off Tot/Pts'].sum()
print("Carolina Panthers Offense Points Mean:", mean_value)
print("Carolina Panthers Offense Points Median:", median_value)
print("Carolina Panthers Offense Points Standard Deviation:", std_value)
print("Carolina Panthers Offense Points Variance:", var_value)
print("Carolina Panthers Offense Points Minimum:", min_value)
print("Carolina Panthers Offense Points Maximum:", max_value)
print("Carolina Panthers Offense Points Range:", data_range)
print("Carolina Panthers Total Points:", total_points)

Carolina Panthers Offense Points Mean: 363.45454545454544
Carolina Panthers Offense Points Median: 357.0
Carolina Panthers Offense Points Standard Deviation: 48.84539617274823
Carolina Panthers Offense Points Variance: 2385.872727272728
Carolina Panthers Offense Points Minimum: 304
Carolina Panthers Offense Points Maximum: 500
Carolina Panthers Offense Points Range: 196
Carolina Panthers Total Points: 3998


### Chicago Bears

In [96]:
team_name6 = 'Chicago Bears'
bears_data = nflstats[nflstats['Team'] == team_name6]
mean_value = bears_data['Off Tot/Pts'].mean()
median_value = bears_data['Off Tot/Pts'].median()
std_value = bears_data['Off Tot/Pts'].std()
var_value = bears_data['Off Tot/Pts'].var()
min_value = bears_data['Off Tot/Pts'].min()
max_value = bears_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = bears_data['Off Tot/Pts'].sum()
print("Chicago Bears Offense Points Mean:", mean_value)
print("Chicago Bears Offense Points Median:", median_value)
print("Chicago Bears Offense Points Standard Deviation:", std_value)
print("Chicago Bears Offense Points Variance:", var_value)
print("Chicago Bears Offense Points Minimum:", min_value)
print("Chicago Bears Offense Points Maximum:", max_value)
print("Chicago Bears Offense Points Range:", data_range)
print("Chicago Bears Total Points:", total_points)

Chicago Bears Offense Points Mean: 324.54545454545456
Chicago Bears Offense Points Median: 319.0
Chicago Bears Offense Points Standard Deviation: 55.69804958230339
Chicago Bears Offense Points Variance: 3102.272727272727
Chicago Bears Offense Points Minimum: 264
Chicago Bears Offense Points Maximum: 445
Chicago Bears Offense Points Range: 181
Chicago Bears Total Points: 3570


### Cincinnati Bengals

In [95]:
team_name7 = 'Cincinnati Bengals'
bengals_data = nflstats[nflstats['Team'] == team_name7]
mean_value = bengals_data['Off Tot/Pts'].mean()
median_value = bengals_data['Off Tot/Pts'].median()
std_value = bengals_data['Off Tot/Pts'].std()
var_value = bengals_data['Off Tot/Pts'].var()
min_value = bengals_data['Off Tot/Pts'].min()
max_value = bengals_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = bengals_data['Off Tot/Pts'].sum()
print("Cincinnati Bengals Offense Points Mean:", mean_value)
print("Cincinnati Bengals Offense Points Median:", median_value)
print("Cincinnati Bengals Offense Points Standard Deviation:", std_value)
print("Cincinnati Bengals Offense Points Variance:", var_value)
print("Cincinnati Bengals Offense Points Minimum:", min_value)
print("Cincinnati Bengals Offense Points Maximum:", max_value)
print("Cincinnati Bengals Offense Points Range:", data_range)
print("Cincinnati Bengals Total Points:", total_points)

Cincinnati Bengals Offense Points Mean: 361.6363636363636
Cincinnati Bengals Offense Points Median: 365.0
Cincinnati Bengals Offense Points Standard Deviation: 65.33647178609009
Cincinnati Bengals Offense Points Variance: 4268.854545454546
Cincinnati Bengals Offense Points Minimum: 279
Cincinnati Bengals Offense Points Maximum: 460
Cincinnati Bengals Offense Points Range: 181
Cincinnati Bengals Total Points: 3978


### Cleveland Browns

In [94]:
team_name8 = 'Cleveland Browns'
browns_data = nflstats[nflstats['Team'] == team_name8]
mean_value = browns_data['Off Tot/Pts'].mean()
median_value = browns_data['Off Tot/Pts'].median()
std_value = browns_data['Off Tot/Pts'].std()
var_value = browns_data['Off Tot/Pts'].var()
min_value = browns_data['Off Tot/Pts'].min()
max_value = browns_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = browns_data['Off Tot/Pts'].sum()
print("Cleveland Browns Offense Points Mean:", mean_value)
print("Cleveland Browns Offense Points Median:", median_value)
print("Cleveland Browns Offense Points Standard Deviation:", std_value)
print("Cleveland Browns Offense Points Variance:", var_value)
print("Cleveland Browns Offense Points Minimum:", min_value)
print("Cleveland Browns Offense Points Maximum:", max_value)
print("Cleveland Browns Offense Points Range:", data_range)
print("Cleveland Browns Total Points:", total_points)

Cleveland Browns Offense Points Mean: 306.54545454545456
Cleveland Browns Offense Points Median: 302.0
Cleveland Browns Offense Points Standard Deviation: 53.94508992737641
Cleveland Browns Offense Points Variance: 2910.0727272727277
Cleveland Browns Offense Points Minimum: 234
Cleveland Browns Offense Points Maximum: 408
Cleveland Browns Offense Points Range: 174
Cleveland Browns Total Points: 3372


### Dallas Cowboys

In [93]:
team_name9 = 'Dallas Cowboys'
cowboys_data = nflstats[nflstats['Team'] == team_name9]
mean_value = cowboys_data['Off Tot/Pts'].mean()
median_value = cowboys_data['Off Tot/Pts'].median()
std_value = cowboys_data['Off Tot/Pts'].std()
var_value = cowboys_data['Off Tot/Pts'].var()
min_value = cowboys_data['Off Tot/Pts'].min()
max_value = cowboys_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = cowboys_data['Off Tot/Pts'].sum()
print("Dallas Cowboys Offense Points Mean:", mean_value)
print("Dallas Cowboys Offense Points Median:", median_value)
print("Dallas Cowboys Offense Points Standard Deviation:", std_value)
print("Dallas Cowboys Offense Points Variance:", var_value)
print("Dallas Cowboys Offense Points Minimum:", min_value)
print("Dallas Cowboys Offense Points Maximum:", max_value)
print("Dallas Cowboys Offense Points Range:", data_range)
print("Dallas Cowboys Total Points:", total_points)

Dallas Cowboys Offense Points Mean: 410.1818181818182
Dallas Cowboys Offense Points Median: 421.0
Dallas Cowboys Offense Points Standard Deviation: 69.50801131066574
Dallas Cowboys Offense Points Variance: 4831.363636363636
Dallas Cowboys Offense Points Minimum: 275
Dallas Cowboys Offense Points Maximum: 530
Dallas Cowboys Offense Points Range: 255
Dallas Cowboys Total Points: 4512


### Denver Broncos

In [92]:
team_name10 = 'Denver Broncos'
broncos_data = nflstats[nflstats['Team'] == team_name10]
mean_value = broncos_data['Off Tot/Pts'].mean()
median_value = broncos_data['Off Tot/Pts'].median()
std_value = broncos_data['Off Tot/Pts'].std()
var_value = broncos_data['Off Tot/Pts'].var()
min_value = broncos_data['Off Tot/Pts'].min()
max_value = broncos_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = broncos_data['Off Tot/Pts'].sum()
print("Denver Broncos Offense Points Mean:", mean_value)
print("Denver Broncos Offense Points Median:", median_value)
print("Denver Broncos Offense Points Standard Deviation:", std_value)
print("Denver Broncos Offense Points Variance:", var_value)
print("Denver Broncos Offense Points Minimum:", min_value)
print("Denver Broncos Offense Points Maximum:", max_value)
print("Denver Broncos Offense Points Range:", data_range)
print("Denver Broncos Total Points:", total_points)

Denver Broncos Offense Points Mean: 369.27272727272725
Denver Broncos Offense Points Median: 333.0
Denver Broncos Offense Points Standard Deviation: 106.46040663936138
Denver Broncos Offense Points Variance: 11333.818181818182
Denver Broncos Offense Points Minimum: 282
Denver Broncos Offense Points Maximum: 606
Denver Broncos Offense Points Range: 324
Denver Broncos Total Points: 4062


### Detroit Lions

In [91]:
team_name11 = 'Detroit Lions'
lions_data = nflstats[nflstats['Team'] == team_name11]
mean_value = lions_data['Off Tot/Pts'].mean()
median_value = lions_data['Off Tot/Pts'].median()
std_value = lions_data['Off Tot/Pts'].std()
var_value = lions_data['Off Tot/Pts'].var()
min_value = lions_data['Off Tot/Pts'].min()
max_value = lions_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = lions_data['Off Tot/Pts'].sum()
print("Detroit Lions Offense Points Mean:", mean_value)
print("Detroit Lions Offense Points Median:", median_value)
print("Detroit Lions Offense Points Standard Deviation:", std_value)
print("Detroit Lions Offense Points Variance:", var_value)
print("Detroit Lions Offense Points Minimum:", min_value)
print("Detroit Lions Offense Points Maximum:", max_value)
print("Detroit Lions Offense Points Range:", data_range)
print("Detroit Lions Total Points:", total_points)

Detroit Lions Offense Points Mean: 373.45454545454544
Detroit Lions Offense Points Median: 372.0
Detroit Lions Offense Points Standard Deviation: 40.77097898349667
Detroit Lions Offense Points Variance: 1662.2727272727273
Detroit Lions Offense Points Minimum: 321
Detroit Lions Offense Points Maximum: 453
Detroit Lions Offense Points Range: 132
Detroit Lions Total Points: 4108


### Green Bay Packers

In [90]:
team_name12 = 'Green Bay Packers'
packers_data = nflstats[nflstats['Team'] == team_name12]
mean_value = packers_data['Off Tot/Pts'].mean()
median_value = packers_data['Off Tot/Pts'].median()
std_value = packers_data['Off Tot/Pts'].std()
var_value = packers_data['Off Tot/Pts'].var()
min_value = packers_data['Off Tot/Pts'].min()
max_value = packers_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = packers_data['Off Tot/Pts'].sum()
print("Green Bay Packers Offense Points Mean:", mean_value)
print("Green Bay Packers Offense Points Median:", median_value)
print("Green Bay Packers Offense Points Standard Deviation:", std_value)
print("Green Bay Packers Offense Points Variance:", var_value)
print("Green Bay Packers Offense Points Minimum:", min_value)
print("Green Bay Packers Offense Points Maximum:", max_value)
print("Green Bay Packers Offense Points Range:", data_range)
print("Green Bay Packers Total Points:", total_points)

Green Bay Packers Offense Points Mean: 407.3636363636364
Green Bay Packers Offense Points Median: 417.0
Green Bay Packers Offense Points Standard Deviation: 62.355870176387924
Green Bay Packers Offense Points Variance: 3888.2545454545448
Green Bay Packers Offense Points Minimum: 320
Green Bay Packers Offense Points Maximum: 509
Green Bay Packers Offense Points Range: 189
Green Bay Packers Total Points: 4481


### Houston Texans

In [89]:
team_name13 = 'Houston Texans'
texans_data = nflstats[nflstats['Team'] == team_name13]
mean_value = texans_data['Off Tot/Pts'].mean()
median_value = texans_data['Off Tot/Pts'].median()
std_value = texans_data['Off Tot/Pts'].std()
var_value = texans_data['Off Tot/Pts'].var()
min_value = texans_data['Off Tot/Pts'].min()
max_value = texans_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = texans_data['Off Tot/Pts'].sum()
print("Houston Texans Offense Points Mean:", mean_value)
print("Houston Texans Offense Points Median:", median_value)
print("Houston Texans Offense Points Standard Deviation:", std_value)
print("Houston Texans Offense Points Variance:", var_value)
print("Houston Texans Offense Points Minimum:", min_value)
print("Houston Texans Offense Points Maximum:", max_value)
print("Houston Texans Offense Points Range:", data_range)
print("Houston Texans Total Points:", total_points)

Houston Texans Offense Points Mean: 335.3636363636364
Houston Texans Offense Points Median: 338.0
Houston Texans Offense Points Standard Deviation: 48.95972370688529
Houston Texans Offense Points Variance: 2397.0545454545454
Houston Texans Offense Points Minimum: 276
Houston Texans Offense Points Maximum: 416
Houston Texans Offense Points Range: 140
Houston Texans Total Points: 3689


### Indianapolis Colts

In [88]:
team_name14 = 'Indianapolis Colts'
colts_data = nflstats[nflstats['Team'] == team_name14]
mean_value = colts_data['Off Tot/Pts'].mean()
median_value = colts_data['Off Tot/Pts'].median()
std_value = colts_data['Off Tot/Pts'].std()
var_value = colts_data['Off Tot/Pts'].var()
min_value = colts_data['Off Tot/Pts'].min()
max_value = colts_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = colts_data['Off Tot/Pts'].sum()
print("Indianapolis Colts Offense Points Mean:", mean_value)
print("Indianapolis Colts Offense Points Median:", median_value)
print("Indianapolis Colts Offense Points Standard Deviation:", std_value)
print("Indianapolis Colts Offense Points Variance:", var_value)
print("Indianapolis Colts Offense Points Minimum:", min_value)
print("Indianapolis Colts Offense Points Maximum:", max_value)
print("Indianapolis Colts Offense Points Range:", data_range)
print("Indianapolis Colts Total Points:", total_points)

Indianapolis Colts Offense Points Mean: 366.1818181818182
Indianapolis Colts Offense Points Median: 361.0
Indianapolis Colts Offense Points Standard Deviation: 73.42045788718316
Indianapolis Colts Offense Points Variance: 5390.563636363637
Indianapolis Colts Offense Points Minimum: 263
Indianapolis Colts Offense Points Maximum: 458
Indianapolis Colts Offense Points Range: 195
Indianapolis Colts Total Points: 4028


### Jacksonville Jaguars

In [87]:
team_name15 = 'Jacksonville Jaguars'
jaguars_data = nflstats[nflstats['Team'] == team_name15]
mean_value = jaguars_data['Off Tot/Pts'].mean()
median_value = jaguars_data['Off Tot/Pts'].median()
std_value = jaguars_data['Off Tot/Pts'].std()
var_value = jaguars_data['Off Tot/Pts'].var()
min_value = jaguars_data['Off Tot/Pts'].min()
max_value = jaguars_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = jaguars_data['Off Tot/Pts'].sum()
print("Jacksonville Jaguars Offense Points Mean:", mean_value)
print("Jacksonville Jaguars Offense Points Median:", median_value)
print("Jacksonville Jaguars Offense Points Standard Deviation:", std_value)
print("Jacksonville Jaguars Offense Points Variance:", var_value)
print("Jacksonville Jaguars Offense Points Minimum:", min_value)
print("Jacksonville Jaguars Offense Points Maximum:", max_value)
print("Jacksonville Jaguars Offense Points Range:", data_range)
print("Jacksonville Jaguars Total Points:", total_points)

Jacksonville Jaguars Offense Points Mean: 322.0
Jacksonville Jaguars Offense Points Median: 306.0
Jacksonville Jaguars Offense Points Standard Deviation: 69.74955196988724
Jacksonville Jaguars Offense Points Variance: 4865.0
Jacksonville Jaguars Offense Points Minimum: 247
Jacksonville Jaguars Offense Points Maximum: 417
Jacksonville Jaguars Offense Points Range: 170
Jacksonville Jaguars Total Points: 3542


### Kansas City Chiefs

In [85]:
team_name16 = 'Kansas City Chiefs'
chiefs_data = nflstats[nflstats['Team'] == team_name16]
mean_value = chiefs_data['Off Tot/Pts'].mean()
median_value = chiefs_data['Off Tot/Pts'].median()
std_value = chiefs_data['Off Tot/Pts'].std()
var_value = chiefs_data['Off Tot/Pts'].var()
min_value = chiefs_data['Off Tot/Pts'].min()
max_value = chiefs_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = chiefs_data['Off Tot/Pts'].sum()
print("Kansas City Chiefs Offense Points Mean:", mean_value)
print("Kansas City Chiefs Offense Points Median:", median_value)
print("Kansas City Chiefs Offense Points Standard Deviation:", std_value)
print("Kansas City Chiefs Offense Points Variance:", var_value)
print("Kansas City Chiefs Offense Points Minimum:", min_value)
print("Kansas City Chiefs Offense Points Maximum:", max_value)
print("Kansas City Chiefs Offense Points Range:", data_range)
print("Kansas City Chiefs Total Points:", total_points)

Kansas City Chiefs Offense Points Mean: 410.72727272727275
Kansas City Chiefs Offense Points Median: 415.0
Kansas City Chiefs Offense Points Standard Deviation: 78.5532824382163
Kansas City Chiefs Offense Points Variance: 6170.618181818182
Kansas City Chiefs Offense Points Minimum: 211
Kansas City Chiefs Offense Points Maximum: 496
Kansas City Chiefs Offense Points Range: 285
Kansas City Chiefs Total Points: 4518


### Las Vegas Raiders

In [84]:
team_name17 = 'Las Vegas Raiders'
raiders_data = nflstats[nflstats['Team'] == team_name17]
mean_value = raiders_data['Off Tot/Pts'].mean()
median_value = raiders_data['Off Tot/Pts'].median()
std_value = raiders_data['Off Tot/Pts'].std()
var_value = raiders_data['Off Tot/Pts'].var()
min_value = raiders_data['Off Tot/Pts'].min()
max_value = raiders_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = raiders_data['Off Tot/Pts'].sum()
print("Las Vegas Raiders Offense Points Mean:", mean_value)
print("Las Vegas Raiders Offense Points Median:", median_value)
print("Las Vegas Raiders Offense Points Standard Deviation:", std_value)
print("Las Vegas Raiders Offense Points Variance:", var_value)
print("Las Vegas Raiders Offense Points Minimum:", min_value)
print("Las Vegas Raiders Offense Points Maximum:", max_value)
print("Las Vegas Raiders Offense Points Range:", data_range)
print("Las Vegas Raiders Total Points:", total_points)

Las Vegas Raiders Offense Points Mean: 341.6363636363636
Las Vegas Raiders Offense Points Median: 322.0
Las Vegas Raiders Offense Points Standard Deviation: 57.69622644033616
Las Vegas Raiders Offense Points Variance: 3328.8545454545456
Las Vegas Raiders Offense Points Minimum: 253
Las Vegas Raiders Offense Points Maximum: 434
Las Vegas Raiders Offense Points Range: 181
Las Vegas Raiders Total Points: 3758


### Los Angeles Chargers

In [83]:
team_name18 = 'Los Angeles Chargers'
chargers_data = nflstats[nflstats['Team'] == team_name18]
mean_value = chargers_data['Off Tot/Pts'].mean()
median_value = chargers_data['Off Tot/Pts'].median()
std_value = chargers_data['Off Tot/Pts'].std()
var_value = chargers_data['Off Tot/Pts'].var()
min_value = chargers_data['Off Tot/Pts'].min()
max_value = chargers_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = chargers_data['Off Tot/Pts'].sum()
print("Los Angeles Chargers Offense Points Mean:", mean_value)
print("Los Angeles Chargers Offense Points Median:", median_value)
print("Los Angeles Chargers Offense Points Standard Deviation:", std_value)
print("Los Angeles Chargers Offense Points Variance:", var_value)
print("Los Angeles Chargers Offense Points Minimum:", min_value)
print("Los Angeles Chargers Offense Points Maximum:", max_value)
print("Los Angeles Chargers Offense Points Range:", data_range)
print("Los Angeles Chargers Total Points:", total_points)

Los Angeles Chargers Offense Points Mean: 374.54545454545456
Los Angeles Chargers Offense Points Median: 355.0
Los Angeles Chargers Offense Points Standard Deviation: 42.924034377871884
Los Angeles Chargers Offense Points Variance: 1842.4727272727273
Los Angeles Chargers Offense Points Minimum: 320
Los Angeles Chargers Offense Points Maximum: 474
Los Angeles Chargers Offense Points Range: 154
Los Angeles Chargers Total Points: 4120


### Los Angeles Rams

In [82]:
team_name19 = 'Los Angeles Rams'
rams_data = nflstats[nflstats['Team'] == team_name19]
mean_value = rams_data['Off Tot/Pts'].mean()
median_value = rams_data['Off Tot/Pts'].median()
std_value = rams_data['Off Tot/Pts'].std()
var_value = rams_data['Off Tot/Pts'].var()
min_value = rams_data['Off Tot/Pts'].min()
max_value = rams_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = rams_data['Off Tot/Pts'].sum()
print("Los Angeles Rams Offense Points Mean:", mean_value)
print("Los Angeles Rams Offense Points Median:", median_value)
print("Los Angeles Rams Offense Points Standard Deviation:", std_value)
print("Los Angeles Rams Offense Points Variance:", var_value)
print("Los Angeles Rams Offense Points Minimum:", min_value)
print("Los Angeles Rams Offense Points Maximum:", max_value)
print("Los Angeles Rams Offense Points Range:", data_range)
print("Los Angeles Rams Total Points:", total_points)

Los Angeles Rams Offense Points Mean: 360.3636363636364
Los Angeles Rams Offense Points Median: 348.0
Los Angeles Rams Offense Points Standard Deviation: 84.86020590037798
Los Angeles Rams Offense Points Variance: 7201.254545454545
Los Angeles Rams Offense Points Minimum: 224
Los Angeles Rams Offense Points Maximum: 478
Los Angeles Rams Offense Points Range: 254
Los Angeles Rams Total Points: 3964


### Miami Dolphins

In [81]:
team_name20 = 'Miami Dolphins'
dolphins_data = nflstats[nflstats['Team'] == team_name20]
mean_value = dolphins_data['Off Tot/Pts'].mean()
median_value = dolphins_data['Off Tot/Pts'].median()
std_value = dolphins_data['Off Tot/Pts'].std()
var_value = dolphins_data['Off Tot/Pts'].var()
min_value = dolphins_data['Off Tot/Pts'].min()
max_value = dolphins_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = dolphins_data['Off Tot/Pts'].sum()
print("Miami Dolphins Offense Points Mean:", mean_value)
print("Miami Dolphins Offense Points Median:", median_value)
print("Miami Dolphins Offense Points Standard Deviation:", std_value)
print("Miami Dolphins Offense Points Variance:", var_value)
print("Miami Dolphins Offense Points Minimum:", min_value)
print("Miami Dolphins Offense Points Maximum:", max_value)
print("Miami Dolphins Offense Points Range:", data_range)
print("Miami Dolphins Total Points:", total_points)

Miami Dolphins Offense Points Mean: 334.1818181818182
Miami Dolphins Offense Points Median: 317.0
Miami Dolphins Offense Points Standard Deviation: 46.96555797990307
Miami Dolphins Offense Points Variance: 2205.7636363636366
Miami Dolphins Offense Points Minimum: 281
Miami Dolphins Offense Points Maximum: 404
Miami Dolphins Offense Points Range: 123
Miami Dolphins Total Points: 3676


### Minnesota Vikings

In [80]:
team_name21 = 'Minnesota Vikings'
vikings_data = nflstats[nflstats['Team'] == team_name21]
mean_value = vikings_data['Off Tot/Pts'].mean()
median_value = vikings_data['Off Tot/Pts'].median()
std_value = vikings_data['Off Tot/Pts'].std()
var_value = vikings_data['Off Tot/Pts'].var()
min_value = vikings_data['Off Tot/Pts'].min()
max_value = vikings_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = vikings_data['Off Tot/Pts'].sum()
print("Minnesota Vikings Offense Points Mean:", mean_value)
print("Minnesota Vikings Offense Points Median:", median_value)
print("Minnesota Vikings Offense Points Standard Deviation:", std_value)
print("Minnesota Vikings Offense Points Variance:", var_value)
print("Minnesota Vikings Offense Points Minimum:", min_value)
print("Minnesota Vikings Offense Points Maximum:", max_value)
print("Minnesota Vikings Offense Points Range:", data_range)
print("Minnesota Vikings Total Points:", total_points)

Minnesota Vikings Offense Points Mean: 385.1818181818182
Minnesota Vikings Offense Points Median: 382.0
Minnesota Vikings Offense Points Standard Deviation: 36.17130957490531
Minnesota Vikings Offense Points Variance: 1308.3636363636363
Minnesota Vikings Offense Points Minimum: 325
Minnesota Vikings Offense Points Maximum: 430
Minnesota Vikings Offense Points Range: 105
Minnesota Vikings Total Points: 4237


### New England Patriots

In [79]:
team_name22 = 'New England Patriots'
patriots_data = nflstats[nflstats['Team'] == team_name22]
mean_value = patriots_data['Off Tot/Pts'].mean()
median_value = patriots_data['Off Tot/Pts'].median()
std_value = patriots_data['Off Tot/Pts'].std()
var_value = patriots_data['Off Tot/Pts'].var()
min_value = patriots_data['Off Tot/Pts'].min()
max_value = patriots_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = patriots_data['Off Tot/Pts'].sum()
print("New England Patriots Offense Points Mean:", mean_value)
print("New England Patriots Offense Points Median:", median_value)
print("New England Patriots Offense Points Standard Deviation:", std_value)
print("New England Patriots Offense Points Variance:", var_value)
print("New England Patriots Offense Points Minimum:", min_value)
print("New England Patriots Offense Points Maximum:", max_value)
print("New England Patriots Offense Points Range:", data_range)
print("New England Patriots Total Points:", total_points)

New England Patriots Offense Points Mean: 442.09090909090907
New England Patriots Offense Points Median: 458.0
New England Patriots Offense Points Standard Deviation: 59.47344709272289
New England Patriots Offense Points Variance: 3537.090909090909
New England Patriots Offense Points Minimum: 326
New England Patriots Offense Points Maximum: 557
New England Patriots Offense Points Range: 231
New England Patriots Total Points: 4863


### New Orleans Saints

In [78]:
team_name23 = 'New Orleans Saints'
saints_data = nflstats[nflstats['Team'] == team_name23]
mean_value = saints_data['Off Tot/Pts'].mean()
median_value = saints_data['Off Tot/Pts'].median()
std_value = saints_data['Off Tot/Pts'].std()
var_value = saints_data['Off Tot/Pts'].var()
min_value = saints_data['Off Tot/Pts'].min()
max_value = saints_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = saints_data['Off Tot/Pts'].sum()
print("New Orleans Saints Offense Points Mean:", mean_value)
print("New Orleans Saints Offense Points Median:", median_value)
print("New Orleans Saints Offense Points Standard Deviation:", std_value)
print("New Orleans Saints Offense Points Variance:", var_value)
print("New Orleans Saints Offense Points Minimum:", min_value)
print("New Orleans Saints Offense Points Maximum:", max_value)
print("New Orleans Saints Offense Points Range:", data_range)
print("New Orleans Saints Total Points:", total_points)

New Orleans Saints Offense Points Mean: 423.5
New Orleans Saints Offense Points Median: 431.0
New Orleans Saints Offense Points Standard Deviation: 49.22567080971743
New Orleans Saints Offense Points Variance: 2423.1666666666665
New Orleans Saints Offense Points Minimum: 330
New Orleans Saints Offense Points Maximum: 482
New Orleans Saints Offense Points Range: 152
New Orleans Saints Total Points: 4235


### New York Giants

In [77]:
team_name24 = 'New York Giants'
giants_data = nflstats[nflstats['Team'] == team_name24]
mean_value = giants_data['Off Tot/Pts'].mean()
median_value = giants_data['Off Tot/Pts'].median()
std_value = giants_data['Off Tot/Pts'].std()
var_value = giants_data['Off Tot/Pts'].var()
min_value = giants_data['Off Tot/Pts'].min()
max_value = giants_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = giants_data['Off Tot/Pts'].sum()
print("New York Giants Offense Points Mean:", mean_value)
print("New York Giants Offense Points Median:", median_value)
print("New York Giants Offense Points Standard Deviation:", std_value)
print("New York Giants Offense Points Variance:", var_value)
print("New York Giants Offense Points Minimum:", min_value)
print("New York Giants Offense Points Maximum:", max_value)
print("New York Giants Offense Points Range:", data_range)
print("New York Giants Total Points:", total_points)

New York Giants Offense Points Mean: 324.45454545454544
New York Giants Offense Points Median: 310.0
New York Giants Offense Points Standard Deviation: 67.00800494920534
New York Giants Offense Points Variance: 4490.072727272727
New York Giants Offense Points Minimum: 246
New York Giants Offense Points Maximum: 429
New York Giants Offense Points Range: 183
New York Giants Total Points: 3569


### New York Jets

In [76]:
team_name25 = 'New York Jets'
Jets_data = nflstats[nflstats['Team'] == team_name25]
mean_value = Jets_data['Off Tot/Pts'].mean()
median_value = Jets_data['Off Tot/Pts'].median()
std_value = Jets_data['Off Tot/Pts'].std()
var_value = Jets_data['Off Tot/Pts'].var()
min_value = Jets_data['Off Tot/Pts'].min()
max_value = Jets_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = Jets_data['Off Tot/Pts'].sum()
print("New York Jets Offense Points Mean:", mean_value)
print("New York Jets Offense Points Median:", median_value)
print("New York Jets Offense Points Standard Deviation:", std_value)
print("New York Jets Offense Points Variance:", var_value)
print("New York Jets Offense Points Minimum:", min_value)
print("New York Jets Offense Points Maximum:", max_value)
print("New York Jets Offense Points Range:", data_range)
print("New York Jets Total Points:", total_points)

New York Jets Offense Points Mean: 294.27272727272725
New York Jets Offense Points Median: 290.0
New York Jets Offense Points Standard Deviation: 35.4008217675548
New York Jets Offense Points Variance: 1253.2181818181818
New York Jets Offense Points Minimum: 243
New York Jets Offense Points Maximum: 387
New York Jets Offense Points Range: 144
New York Jets Total Points: 3237


### Philadelphia Eagles

In [75]:
team_name26 = 'Philadelphia Eagles'
eagles_data = nflstats[nflstats['Team'] == team_name26]
mean_value = eagles_data['Off Tot/Pts'].mean()
median_value = eagles_data['Off Tot/Pts'].median()
std_value = eagles_data['Off Tot/Pts'].std()
var_value = eagles_data['Off Tot/Pts'].var()
min_value = eagles_data['Off Tot/Pts'].min()
max_value = eagles_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = eagles_data['Off Tot/Pts'].sum()
print("Philadelphia Eagles Offense Points Mean:", mean_value)
print("Philadelphia Eagles Points Median:", median_value)
print("Philadelphia Eagles Points Standard Deviation:", std_value)
print("Philadelphia Eagles Points Variance:", var_value)
print("Philadelphia Eagles Points Minimum:", min_value)
print("Philadelphia Eagles Points Maximum:", max_value)
print("Philadelphia Eagles Points Range:", data_range)
print("Philadelphia Eagles Total Points:", total_points)

Philadelphia Eagles Offense Points Mean: 408.54545454545456
Philadelphia Eagles Points Median: 442.0
Philadelphia Eagles Points Standard Deviation: 64.3418427407292
Philadelphia Eagles Points Variance: 4139.872727272728
Philadelphia Eagles Points Minimum: 280
Philadelphia Eagles Points Maximum: 477
Philadelphia Eagles Points Range: 197
Philadelphia Eagles Total Points: 4494


### Pittsburgh Steelers

In [74]:
team_name27 = 'Pittsburgh Steelers'
steelers_data = nflstats[nflstats['Team'] == team_name27]
mean_value = steelers_data['Off Tot/Pts'].mean()
median_value = steelers_data['Off Tot/Pts'].median()
std_value = steelers_data['Off Tot/Pts'].std()
var_value = steelers_data['Off Tot/Pts'].var()
min_value = steelers_data['Off Tot/Pts'].min()
max_value = steelers_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = steelers_data['Off Tot/Pts'].sum()
print("Pittsburgh Steelers Offense Points Mean:", mean_value)
print("Pittsburgh Steelers Points Median:", median_value)
print("Pittsburgh Steelers Points Standard Deviation:", std_value)
print("Pittsburgh Steelers Points Variance:", var_value)
print("Pittsburgh Steelers Points Minimum:", min_value)
print("Pittsburgh Steelers Points Maximum:", max_value)
print("Pittsburgh Steelers Points Range:", data_range)
print("Pittsburgh Steelers Total Points:", total_points)

Pittsburgh Steelers Offense Points Mean: 376.45454545454544
Pittsburgh Steelers Points Median: 399.0
Pittsburgh Steelers Points Standard Deviation: 49.62532344753762
Pittsburgh Steelers Points Variance: 2462.672727272727
Pittsburgh Steelers Points Minimum: 289
Pittsburgh Steelers Points Maximum: 436
Pittsburgh Steelers Points Range: 147
Pittsburgh Steelers Total Points: 4141


### San Francisco 49ers

In [73]:
team_name28 = 'San Francisco 49ers'
niners_data = nflstats[nflstats['Team'] == team_name28]
mean_value = niners_data['Off Tot/Pts'].mean()
median_value = niners_data['Off Tot/Pts'].median()
std_value = niners_data['Off Tot/Pts'].std()
var_value = niners_data['Off Tot/Pts'].var()
min_value = niners_data['Off Tot/Pts'].min()
max_value = niners_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = niners_data['Off Tot/Pts'].sum()
print("San Francisco 49ers Offense Points Mean:", mean_value)
print("San Francisco 49ers Points Median:", median_value)
print("San Francisco 49ers Points Standard Deviation:", std_value)
print("San Francisco 49ers Points Variance:", var_value)
print("San Francisco 49ers Points Minimum:", min_value)
print("San Francisco 49ers Points Maximum:", max_value)
print("San Francisco 49ers Points Range:", data_range)
print("San Francisco 49ers Total Points:", total_points)

San Francisco 49ers Offense Points Mean: 368.1818181818182
San Francisco 49ers Points Median: 376.0
San Francisco 49ers Points Standard Deviation: 71.99835856714814
San Francisco 49ers Points Variance: 5183.763636363636
San Francisco 49ers Points Minimum: 238
San Francisco 49ers Points Maximum: 479
San Francisco 49ers Points Range: 241
San Francisco 49ers Total Points: 4050


### Tampa Bay Buccaneers

In [72]:
team_name29 = 'Tampa Bay Buccaneers'
buccaneers_data = nflstats[nflstats['Team'] == team_name29]
mean_value = buccaneers_data['Off Tot/Pts'].mean()
median_value = buccaneers_data['Off Tot/Pts'].median()
std_value = buccaneers_data['Off Tot/Pts'].std()
var_value = buccaneers_data['Off Tot/Pts'].var()
min_value = buccaneers_data['Off Tot/Pts'].min()
max_value = buccaneers_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = buccaneers_data['Off Tot/Pts'].sum()
print("Tampa Bay Buccaneers Offense Points Mean:", mean_value)
print("Tampa Bay Buccaneers Points Median:", median_value)
print("Tampa Bay Buccaneers Points Standard Deviation:", std_value)
print("Tampa Bay Buccaneers Points Variance:", var_value)
print("Tampa Bay Buccaneers Points Minimum:", min_value)
print("Tampa Bay Buccaneers Points Maximum:", max_value)
print("Tampa Bay Buccaneers Points Range:", data_range)
print("Tampa Bay Buccaneers Total Points:", total_points)

Tampa Bay Buccaneers Offense Points Mean: 372.1818181818182
Tampa Bay Buccaneers Points Median: 342.0
Tampa Bay Buccaneers Points Standard Deviation: 80.57892799214716
Tampa Bay Buccaneers Points Variance: 6492.963636363637
Tampa Bay Buccaneers Points Minimum: 277
Tampa Bay Buccaneers Points Maximum: 511
Tampa Bay Buccaneers Points Range: 234
Tampa Bay Buccaneers Total Points: 4094


### Tennessee Titans

In [71]:
team_name30 = 'Tennessee Titans'
titans_data = nflstats[nflstats['Team'] == team_name30]
mean_value = titans_data['Off Tot/Pts'].mean()
median_value = titans_data['Off Tot/Pts'].median()
std_value = titans_data['Off Tot/Pts'].std()
var_value = titans_data['Off Tot/Pts'].var()
min_value = titans_data['Off Tot/Pts'].min()
max_value = titans_data['Off Tot/Pts'].max()
data_range = max_value - min_value
total_points = titans_data['Off Tot/Pts'].sum()
print("Tennessee Titans Offense Points Mean:", mean_value)
print("Tennessee Titans Points Median:", median_value)
print("Tennessee Titans Points Standard Deviation:", std_value)
print("Tennessee Titans Points Variance:", var_value)
print("Tennessee Titans Points Minimum:", min_value)
print("Tennessee Titans Points Maximum:", max_value)
print("Tennessee Titans Points Range:", data_range)
print("Tennessee Titans Total Offense Points:", total_points)

Tennessee Titans Offense Points Mean: 354.90909090909093
Tennessee Titans Points Median: 334.0
Tennessee Titans Points Standard Deviation: 66.0446130209793
Tennessee Titans Points Variance: 4361.890909090909
Tennessee Titans Points Minimum: 254
Tennessee Titans Points Maximum: 491
Tennessee Titans Points Range: 237
Tennessee Titans Total Offense Points: 3904


### Washington Commanders

In [103]:
team_name31 = 'Washington Commanders'
commanders_data = nflstats[nflstats['Team'] == team_name31]
mean_value = commanders_data['Off Tot/Pts'].mean()
median_value = commanders_data['Off Tot/Pts'].median()
std_value = commanders_data['Off Tot/Pts'].std()
var_value = commanders_data['Off Tot/Pts'].var()
min_value = commanders_data['Off Tot/Pts'].min()
max_value = commanders_data['Off Tot/Pts'].max()
total_points = commanders_data['Off Tot/Pts'].sum()
data_range = max_value - min_value
print("Washington Commanders Offense Points Mean:", mean_value)
print("Washington Commanders Median:", median_value)
print("Washington Commanders Standard Deviation:", std_value)
print("Washington Commanders Variance:", var_value)
print("Washington Commanders Minimum:", min_value)
print("Washington Commanders Points Maximum:", max_value)
print("Washington Commanders Total Offense Points:", total_points)
print("Washington Commanders Points Range:", data_range)

Washington Commanders Offense Points Mean: 345.09090909090907
Washington Commanders Median: 335.0
Washington Commanders Standard Deviation: 46.714996618761624
Washington Commanders Variance: 2182.2909090909097
Washington Commanders Minimum: 266
Washington Commanders Points Maximum: 436
Washington Commanders Total Offense Points: 3796
Washington Commanders Points Range: 170


#### Analysis Findings

#### New England Patriots stand out with highest average of total offense points per year @ 443 pts per year average. NE also is the team that has the maximum number of points in a year with 557 pts in 2012.

#### New York Jets stand out with lowest average of total offense points per year @ 294 pts per year average. NYJ is the team that has the lowest number of points in a year with 243 in 2020.

#### Seattle Seahawks stand out with the lowest standard deviation of pts and lowest range of points. The Denver Broncos have the highest standard deviation and highets range of points. 